modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Models/PublisherExtensions.cs

```diff
@@ -41,7 +41,7 @@ public static class PublisherExtensions {
                 DataSetClassId = model.DataSetClassId,
                 DataSetDescription = model.DataSetDescription,
                 DataSetKeyFrameCount = model.DataSetKeyFrameCount,
-                DataSetMetaDataSendInterval = model.DataSetMetaDataSendInterval,
+                MetaDataUpdateTime = model.DataSetMetaDataSendInterval,
```

Comment:
I'm not sure if we're missing the `MetaDataQueueName` here?

Reply:
Totally correct.  Will fix